project/Dependencies.scala

```diff
@@ -62,6 +64,7 @@ object Dependencies {
     "ch.qos.logback" % "logback-classic" % "1.1.3",
     "org.apache.commons" % "commons-lang3" % "3.4",
     "com.typesafe" % "config" % "1.3.0",
+    "org.codehaus.janino" % "janino" % "2.7.8",
```

Comment:
I'll remove akka dependency, I was trying to add the Retry in the database layer itself but since it needs ActorSystem and I didn't want to add akka dependency, I moved it to core.
This `"org.codehaus.janino" % "janino" % "2.7.8`" is to allow "if" statements in logback.xml, which I copied over to the database project to avoid all the noise from slick. This particular one in core is again a breadcrumb from previous attempts which I'll nix. I'll harmonize the version.